src/BaseCurve.sol

```diff
@@ -55,19 +68,19 @@ abstract contract BaseCurve {
     /// @param totalShares Total quantity of shares already awarded by the curve
     /// @return shares The number of shares that would need to be redeemed
     function previewWithdraw(uint256 assets, uint256 totalAssets, uint256 totalShares)
-        public
```

Comment:
3.20 - Functions in Curve Contracts Could Be external Instead of public

Made this `external`